src/hooks/utils.c

```diff
@@ -0,0 +1,29 @@
+#include <postgres.h>
+
+#include <catalog/pg_type_d.h>
+#include <nodes/makefuncs.h>
+#include <nodes/pg_list.h>
+
+List *get_operator_oids()
+{
+    List *oidList = NIL;
+
+    Oid intArrayOid = INT4ARRAYOID;
```

Comment:
I'd use these constants directly in the function call below, no need for the local variables. but this is fine too

Reply:
Done. This was a relic of an older version of the code.